src/prsdk/prescriptors/prescriptor.py

```diff
@@ -6,6 +6,7 @@
 import pandas as pd
 
 
+# pylint: disable=too-few-public-methods
```

Comment:
Added this because as of now we only need our prescriptor interface to have the single public method